vsts_cd_manager/continuous_delivery_manager.py

```diff
@@ -106,18 +119,13 @@ def setup_continuous_delivery(self, azure_deployment_slot, app_type, vsts_accoun
         account_created = False
         accountClient = Account('3.2-preview', None, self._azure_info.credentials)
         if create_account:
-            # Try to create the account (already existing accounts are fine too)
-            self._update_progress(0, 100, 'Creating or getting Team Services account information')
-            properties = {}
-            #TODO right now it is hard to match a random Azure region to a VSTS region
-            #properties['Microsoft.VisualStudio.Services.Account.TfsAccountRegion'] = self._azure_info.webapp_location
-            properties['Microsoft.VisualStudio.Services.Account.SignupEntryPoint'] = 'AzureCli'
-            account_creation_parameters = AccountCreateInfoInternal(
-                vsts_account_name, None, vsts_account_name, None, properties)
-            creation_results = accountClient.create_account(account_creation_parameters, True)
-            account_created = not creation_results.account_id == None
-            if account_created:
-                self._update_progress(5, 100, 'Team Services account created')
+            file_path = os.path.join(os.path.dirname(__file__), 'template.json')
+            deployment_name = 'VSTS.Account.Creation'
+            deployment_mode='incremental'
+            no_wait=False
+            parameters = self.get_parameters(vsts_account_name)
+            deploy_arm_template(self._azure_info.resource_group_name, file_path, None, deployment_name, parameters, deployment_mode, no_wait)
+            self._update_progress(5, 100, 'Team Services account created')
```

Comment:
This code path should also work if the account already exists. Make sure to test that case. If you can detect that case, you may want to change the progress message to say created only if it was actually created.